### PR TITLE
Removed dot from substring

### PR DESCRIPTION
**Pull request by Andrei-Predoiu**
https://github.com/shibayan/keyvault-acmebot/issues/184

### CODE REVIEW

**Review by shibayan — CHANGES_REQUESTED** [review]
Focus on the code formatting before you modify it. Essentially, it should only take one line of change.

**Comment by Andrei-Predoiu** [comment]
> 
> 
> Focus on the code formatting before you modify it. Essentially, it should only take one line of change.

My setup has some custom formatting rules, my bad

File: KeyVault.Acmebot/Providers/GoogleDnsProvider.cs [hunk 1]
```diff
@@ -3,13 +3,11 @@
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
-
 using Google.Apis.Auth.OAuth2;
 using Google.Apis.Dns.v1;
 using Google.Apis.Dns.v1.Data;
 using Google.Apis.Json;
 using Google.Apis.Services;
-
```
Comment (shibayan) [inline KeyVault.Acmebot/Providers/GoogleDnsProvider.cs]: Restore linebreak

File: KeyVault.Acmebot/Providers/GoogleDnsProvider.cs [hunk 2]
```diff
@@ -3,13 +3,11 @@
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
-
```
Comment (shibayan) [inline KeyVault.Acmebot/Providers/GoogleDnsProvider.cs]: Restore linebreak

File: KeyVault.Acmebot/Providers/GoogleDnsProvider.cs [hunk 3]
```diff
@@ -35,9 +33,7 @@ public async Task<IReadOnlyList<DnsZone>> ListZonesAsync()
         {
             var zones = await _dnsService.ManagedZones.List(_credsParameters.ProjectId).ExecuteAsync();
 
-            return zones.ManagedZones
-                        .Select(managedZone => new DnsZone { Id = managedZone.Id.ToString(), Name = managedZone.DnsName })
-                        .ToArray();
+            return zones.ManagedZones.Select(managedZone => new DnsZone { Id = managedZone.Id.ToString(), Name = managedZone.DnsName.Remove(managedZone.DnsName.Length - 1) }).ToArray();
```
Comment (shibayan) [inline KeyVault.Acmebot/Providers/GoogleDnsProvider.cs]:
```suggestion
            return zones.ManagedZones
                        .Select(managedZone => new DnsZone { Id = managedZone.Id.ToString(), Name = managedZone.DnsName.Remove(managedZone.DnsName.Length - 1) })
                        .ToArray();
```

**Review by shibayan — CHANGES_REQUESTED** [review]
The formatting still needs to be improved for consistency with existing code. Recommended code suggestions have been added.

**Review by shibayan — APPROVED** [review]
LGTM. Thanks.

**Pull request merged**